db/migrate/20191203155817_add_allow_password_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddAllowPasswordToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :allow_password_change, :boolean, default: false, null: false
```

Comment:
Why was this necessary?

Reply:
Because devise need it, devise set the value on true when the user ask for a reset token and check the boolean value before update the password.